mycroft/res/text/en-us/configurations.json

```diff
@@ -0,0 +1,33 @@
+{
+  "unit system": {
```

Comment:
So if I've understood correctly, the following Feature file should work
```
Feature: Mycroft Weather Skill local forecasts and temperature

  Scenario Outline: Temperature reported in metric
    Given an english speaking user
    Given the user's "unit system" is "metric"
     When the user says "<current local weather>"
     Then mycroft reply should contain "celsius"

  Examples: What is the current local weather
    | current local weather |
    | tell me the current weather |
    | what's the current weather like |
```
 however I'm getting a NoneType not iterable error:
```
    Given the user's "unit system" is "metric"                                                # features/steps/configuration.py:98 0.000s
      Traceback (most recent call last):
        File "/home/kris/mycroft-core/.venv/lib/python3.7/site-packages/behave/model.py", line 1806, in run
          match.run(runner.context)
        File "/home/kris/mycroft-core/.venv/lib/python3.7/site-packages/behave/matchers.py", line 97, in run
          self.func(context, *args, **kwargs)
        File "features/steps/configuration.py", line 103, in given_config
          patch_config(context, patch_dict)
        File "features/steps/configuration.py", line 32, in patch_config
          for key in patch:
      TypeError: 'NoneType' object is not iterable

```
Any ideas?

Reply:
Initially the quotes shouldn't be there but let me verify.

Reply:
Without the quotes it tests atleast but the test isn't really deterministic since it rarely states the units.

Reply:
I'll add so it allows both with and without `"`

Reply:
Done